test/pre-test-scripts.ts

```diff
@@ -96,7 +96,7 @@ async function seedOrganizations() {
 }
 
 async function relateOrganizationsToTokens() {
-  const tokens = await Token.find({});
+  const tokens = await Token.createQueryBuilder('token').getMany();
```

Comment:
Since I added the eager:true to the entity. This was not allowing the tokens to save with the organization because of the null loaded org.
We have to be careful when using find() methods, as they are the only ones affected by cascade: true, eager: true and other similar flags. 
createQueryBuilder does not call those methods. They are useful but might damage our tests.

Reply:
Just saying it to clarify, because in most/all of our mutations/queries we use always queryBuilder.
And I think the queryBuilder is 100x more powerful than the entity find({}) methods, but we can use them being careful.

Reply:
Good point, I didn't know about the performance difference, thanks for mentioning that, I love to hear these type of advice

Reply:
BTW I prefer using raw queries because it's easier to copy those queries and test them in Db viewer applications (like pg admin), and its easier to find solutions in Postgres query instead of finding solutions in TypeOrm